### DISCUSSION TITLE
Server Error 500 when accese the README.md file in my repo

### DISCUSSION THREAD

**Discussion by andycall** [community/community#76599] [Repositories]
### Select Topic Area

Bug

### Body

Github return 500 error when accessing the following URL:

https://github.com/openwebf/webf/blob/main/README.md

<img width="1786" alt="image" src="https://github.com/community/community/assets/4409743/af184b84-93b1-4bc7-b7da-884abda838c0">

And the README preview also crashed:

<img width="1058" alt="image" src="https://github.com/community/community/assets/4409743/fa7727ff-f930-4a9b-969b-a3633142b70f">

**Comment by cmkobel** [comment]
Exactly same problem at my https://github.com/cmkobel/assemblycomparator2

**Comment by andycall** [comment]
Recovered

**Reply by cmkobel** [reply]
Yeah looks like it.
